reference/docs-conceptual/windows-powershell/whats-new/What-s-New-in-Windows-PowerShell-50.md

```diff
@@ -111,10 +111,10 @@ PowerShell 4.0 are available in the
 
 ### New features in Windows PowerShell
 
-- Starting in Windows PowerShell 5.0, you can develop by using classes, by using formal syntax and
-  semantics that are similar to other object-oriented programming languages. **Class**, **Enum**,
-  and other keywords have been added to the Windows PowerShell language to support the new feature.
-  For more information about working with classes, see about_Classes.
+- Starting in Windows PowerShell 5.0, you can develop using classes, formal syntax, and semantics
+  that are similar to other object-oriented programming languages. **Class**, **Enum**, and other
+  keywords have been added to the Windows PowerShell language to support the new feature. For more
+  information about working with classes, see about_Classes.
```

Comment:
Should this link to the document directly?
```suggestion
  information about working with classes, see about_Classes.
```

Reply:
There are so many of these kinds of references in this doc. Don't want to fix them in this PR. May not fix them ever since this is so outdated.